apps/deploy-web/src/components/new-deployment/CreateLease.tsx

```diff
@@ -467,6 +506,31 @@ export const CreateLease: React.FunctionComponent<Props> = ({ dseq }) => {
           )}
         </ViewPanel>
       )}
+
+      {zeroBidsForTrialWarningDisplayed && (
+        <div className="pt-4">
+          <Card>
+            <CardContent>
+              <div className="px-16 pb-4 pt-6 text-center">
+                <h3 className="mb-4 text-xl font-bold">Waiting for bids</h3>
+                <p className="mb-8">
+                  It looks like you’re not receiving any bids. This is likely because all trial providers are currently in use. Console offers{" "}
+                  {trialProviderCount} providers for trial users, but many more are available for non-trial users. To access the full list of providers, we
+                  recommend signing up.
```

Comment:
```suggestion
                  It looks like you’re not receiving any bids. This is likely because all trial providers are currently in use. Console offers{" "}
                  {trialProviderCount} providers for trial users, but many more are available for non-trial users. To access the full list of providers, we recommend signing up and adding funds to your account.
```